README.md

```diff
@@ -366,6 +373,18 @@ Code of Conduct - If anyone is reported as malicious or making others feel uncom
 
 ### US
 
+###Europe
+
+##### Oana Galbenu 
+- [@oanaalex](https://twitter.com/oanaalex)
+- [galbenu.ro](http://galbenu.ro/)
+- oana@galbenu.ro. oanagalbenu@gmail.com
+- Topics of Interest- HTML, CSS and preprocessors, Agile
+- Location - Bucharest, Romania, other Romanian cities
+- Languages besides English - Romanian
+- I would like to get involved because lately in Romania there are more and more developers that forgot why the web is here. They do not know the basic rules and they are in serious need of guidance. I would like to put together a group of exepienced people that would volunteer to mentor and guide the emergind developers and help them on the right track.
```

Comment:
Can you please revise this entry to what you would like to offer the community? We would like to refrain from negative language regarding any particular group. Thank you!

Reply:
Changed and pushed again. Thank you!